fs/fstest/file.go

```diff
@@ -12,6 +12,7 @@ import (
 // Applier applies single file changes
 type Applier interface {
 	Apply(root string) error
+	Verify(root string) error
```

Comment:
I don't think this interface is correct. An applier does not have the ability to isolate it's change in a way to properly verify a change was applied. This would require passing in both a base directory and a changed directory. However doing this does not belong in the interface and better left to a helper.

For further context, the implemented `Verify` function is only valid for an applier that works on a root, but every incremental Applier has an invalid Verify function.